uint64 timestamp_utc # UTC timestamp

uint32 seq		# Image sequence number
